docs/application/tizen-studio/setup/prerequisites.md

```diff
@@ -186,7 +186,7 @@ The following table lists the additional requirements to be met before developin
 <tr>
 <td>Python for using the T-trace (Tizen profiling tool used to optimize the application performance)
 </td>
-<td>To use the T-trace in Windows&reg; 8/7, you must install a Python 2.7.X version:
+<td>To use the T-trace in Windows, you must install a Python 2.7.X version:
```

Comment:
@mijin85cho spoke to Karthik, he suggested to mention like this:

To use the T-trace in Windows&reg; 10 and above, you must install a Python 2.7.X version:

Reply:
Question.

Isn't the "least" version for the Tizen Studio itself the Windows 10?
Thus there is no other choice except "Windows 10 and above" for now, so I didn't mention in previous update.

![image](https://user-images.githubusercontent.com/9958374/91937624-467b0500-ed2d-11ea-9f4e-ef23dffe0389.png)

Reply:
@annie-samsung Would you please let me know you opinion to my comment?

Reply:
@mijin85cho the idea here was to keep the document future proof, so that we don't have to update going forward for next Windows version. However, if you want we can just mention Windows 10 for now, and remember to change it when another Windows version is also supported.

Reply:
@annie-samsung It is a good idea to prepare the future version, but I think it is better to change in that way if the Tizen Studio "really" supports the future version. For now, "the future version" doesn't exist, and I can't sure that Tizen Studio will support that version from when. (Maybe it supports immediately when future version is released. But I think it also has the possibility that further working is needed to make Tizen Studio compatible with the "above Windows 10". I'm not sure with it.)